spec/reporter_spec.rb

```diff
@@ -154,6 +154,43 @@ def self.method_missing(m, *args, &block)
     end
   end
 
+  context "with a SentryNotifier" do
+    let(:sentry_options) do
+      {
+        "dsn" => 'https://123abc:def456@app.getsentry.com/98765',
+        "environments" => ['production', 'staging'],
+        "current_environment" => 'production'
+      }
+    end
+
+    before(:each) do
+      Adhearsion::Reporter.config.notifier = Adhearsion::Reporter::SentryNotifier
+      Adhearsion::Reporter.config.sentry = sentry_options
+    end
+
+    it "should initialize correctly" do
+      Adhearsion::Plugin.init_plugins
+
+      config = Raven.configuration
+      sentry_options.each do |k, v|
```

Comment:
Do the other tests use an each loop? Otherwise I'd probably just write it out, splitting the URL comparison into its own test.